src/cli/README.md

```diff
@@ -1144,6 +1144,20 @@ Set the customized data poll period for sleepy end device (milliseconds >= 10ms)
 Done
 ```
 
+### pskc
+
+Usage: `pskc [-p] <key>|<passphrase>`
+
+Set pskc with hex format or passphrase if `-p` is given when the pskc is
```

Comment:
"Commissioner Credential" term should be explicitly mentioned here. And UTF-8 format specified.